packages/component-library/stories/Icons.ts

```diff
@@ -0,0 +1,379 @@
+// LOGO
+import caMonogram from "@kaizen/component-library/icons/ca-monogram.icon.svg"
+
+// ACTIONS
+import draft from "@kaizen/component-library/icons/writing.icon.svg"
+import edit from "@kaizen/component-library/icons/edit.icon.svg"
+import trash from "@kaizen/component-library/icons/trash.icon.svg"
+import refresh from "@kaizen/component-library/icons/refresh.icon.svg"
+import sync from "@kaizen/component-library/icons/sync.icon.svg"
+import duplicate from "@kaizen/component-library/icons/duplicate.icon.svg"
+import undo from "@kaizen/component-library/icons/undo.icon.svg"
+import redo from "@kaizen/component-library/icons/redo.icon.svg"
+import send from "@kaizen/component-library/icons/send.icon.svg"
+import print from "@kaizen/component-library/icons/print.icon.svg"
+import configure from "@kaizen/component-library/icons/configure.icon.svg"
+import flagOn from "@kaizen/component-library/icons/flag-on.icon.svg"
```

Comment:
It's going to be great to have the icons in Storybook, but I think we can make it even easier for an engineer to import the icon that they need.

Look at the `flagOff` icon. How would the engineer know to import it like so?
```
import flagOffIcon from "@kaizen/component-library/icons/flag-off.icon.svg"
```
<img width="549" alt="image" src="https://user-images.githubusercontent.com/763385/197697135-e879bed1-0e89-431d-a932-69c831fcc265.png">

As an idea, could we put some info at the top of the stickersheet saying something like:
```
To import an icon use this import statement and replace the file name with the name under the icon:
import myIcon from "@kaizen/component-library/icons/{FILE_NAME}.icon.svg"
```

Reply:
Perhaps another thing we could do is [copy the icon name to their clipboard](https://www.npmjs.com/package/copy-to-clipboard) when clicked

Reply:
@gyfchong That's a great idea. I've create [a card](https://cultureamp.atlassian.net/jira/software/projects/KDS/boards/289?selectedIssue=KDS-916) to track it, as I think it'd be helpful for all docs we have.